server/database/models/User.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+
+const UserSchema = new mongoose.SchemaType({
+  email: {
+    type: String,
+    default: '',
+  },
+  password: {
+    type: String,
+    default: '',
+  },
+  name: {
+    type: String,
+    default: '',
+  },
+  isDeleted: {
+    type: Boolean,
+    default: false,
+  },
+  signUpDate: {
+    type: Date,
```

Comment:
for what we gonna need this information ?
SignUpDate?

Reply:
Good question we will use it for the timeline! The first entry :)